Website: add documentation for `tag` and `titleTag` arguments (#2365)
hashicorp/design-system: website/docs/components/code-block/partials/code/how-to-use.md

</Hds::CodeBlock>
```

### Title `tag`

> The `tag` value changes the HTML element wrapping the content in the title block. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if a CodeBlock is within a subsection of the page below a heading level 2, the value should be `"h3"`. 

Reviewer: is it `tag` or `titleTag`?  I'm asking because the matching component api says titleTag.

Author: Oops, its `tag`

> The `tag` value changes the HTML element wrapping the content in the title block. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if a CodeBlock is within a subsection of the page below a heading level 2, the value should be `"h3"`. 

Reviewer 2: ```suggestion
The `@tag` attribute changes the HTML element that wraps the `CodeBlock::Title` content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if a CodeBlock is within a subsection of the page below a heading level 2, the value should be `"h3"`. 
```
